### PR TITLE
treepl 2017.05.23

### PR DESCRIPTION
**Pull request by jonchang**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

File: Formula/treepl.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  homepage "https://github.com/blackrim/treePL/wiki"
+  url "https://github.com/blackrim/treePL/archive/fcddad99f4ecbada67ecd210f12fefa762f0d51c.tar.gz"
+  version "2017.05.23"
+  sha256 "7543dbf3cc174bc4b7a39618267439c11c9cd5d1f18ef5b42f0159f389684370"
+  head "https://github.com/blackrim/treePL.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "gcc"
```
Comment (sjackman) [inline Formula/treepl.rb]: Please change this line like so and move it below `depends_on "nlopt"`
```ruby
depends_on "gcc" if OS.mac? # needs openmp
```

File: Formula/treepl.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        system "make", "install"
+      end
+    end
+
+    cd "src" do
+      # Tell configure where to find libraries
+      nlopt = Formula["nlopt"]
+      ENV.prepend "CPPFLAGS", "-I#{libexec}/include"
+      ENV.append "CPPFLAGS", "-I#{nlopt.include}"
+      ENV.append "CFLAGS", "-I#{nlopt.include}"
```
Comment (sjackman) [inline Formula/treepl.rb]: You can use `ENV.append_to_cflags` to update both `CFLAGS` and `CPPFLAGS`
```ruby
ENV.append_to_cflags "-I#{nlopt.include}"
```

**Comment by jonchang** [comment]
Changes made, CI is 💚

**Comment by sjackman** [comment]
CircleCI didn't run for some reason. Can you please rebase this PR onto `master`?
I wish that GitHub had a button for that, to rebase and rerun CI.

File: Formula/treepl.rb (hunk 1 above)
Comment (jonchang) [reply Formula/treepl.rb]: This caused audit failure:

```
brewsci/bio/treepl:
  * C: 14: col 3: dependency "gcc" (line 14) should be put before dependency "nlopt" (line 13)
```

so I will reorder this to make audit happy

**Comment by sjackman** [comment]
Thanks, Jon! Welcome to Brewsci/bio! 🎆

**Pull request closed without merging**